src/python/pants/backend/python/rules/pex.py

```diff
@@ -29,7 +36,7 @@ class RequirementsPex(datatype([('directory_digest', Digest)])):
 
 # TODO: This is non-hermetic because the requirements will be resolved on the fly by
 # pex, where it should be hermetically provided in some way.
-@rule(RequirementsPex, [RequirementsPexRequest, DownloadedPexBin, PythonSetup, PexBuildEnvironment, Platform])
+@rule(RequirementsPex, [MakePexRequest, DownloadedPexBin, PythonSetup, PexBuildEnvironment, Platform])
 def create_requirements_pex(request, pex_bin, python_setup, pex_build_environment, platform):
```

Comment:
Should rename to something like `create_pex`.

Reply:
+1, and `s/RequirementsPex/Pex/` or some other more generic name than `RequirementsPex`.